src/pages/Likes/Likes.jsx

```diff
@@ -38,10 +46,16 @@ const Likes = () => {
             );
           })
         ) : (
-          <p className={styles.noLikes}>You haven't liked any images yet...</p>
+          <p className={styles.noLikes}>
+            You haven&apos;t liked any images yet...
+          </p>
         )}
       </div>
 
+      <button onClick={handleClear} className={styles.clearButton}>
+        Clear Likes
+      </button>
```

Comment:
This works but doesn't fit the UI. 

There is however a Button component in the project that you can import and use here. The classname clearButton will not be needed since the Button component is styled already. 

Also, in your implementation the button is visible even if there are no images to clear. If there are no favorites in local storage the button shouldn't be visible for the user.

A solution would be to conditionally render the button based on if `favorites` (`const [favorites, setFavorites] = useState(getFavoritesFromLocalStorage());` is empty or not.

If you could make these changes I will merge your PR. 

Thanks!

Reply:
I'll do these changes.